zh/unity-asr.md

```diff
@@ -0,0 +1,369 @@
+---
+title: "如何在 Unity 游戏中集成 AI 语音识别？"
+thumbnail: /blog/assets/124_ml-for-games/unity-asr-thumbnail.png
+authors:
+- user: dylanebert
+translators:
+- user: SuSung-boy
+---
+
+<h1>如何在 Unity 游戏中集成 AI 语音识别？</h1>
+
+<!-- {blog_metadata} -->
+<!-- {authors} -->
+
+[![Open Source AI Game Jam](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/124_ml-for-games/gamejambanner.png)](https://itch.io/jam/open-source-ai-game-jam)
+
+## 简介
+
+语音识别是一项将语音转换为文本的技术，想象一下它如何在游戏中发挥作用？发出命令操纵控制面板或者游戏角色、直接与 NPC 对话、提升交互性等等，都有可能。本文将介绍如何使用 Hugging Face Unity API 在 Unity 游戏中集成 SOTA 语音识别功能。
+
+您可以访问 [itch.io 网站](https://individualkex.itch.io/speech-recognition-demo) 下载 Unity 游戏样例，亲自尝试一下语音识别功能。
+
+### 先决条件
+
+阅读文本可能需要了解一些 Unity 的基本概念。除此之外，您还需安装 [Hugging Face Unity API](https://github.com/huggingface/unity-api)，可以点击 [之前的博文](https://huggingface.co/blog/unity-api) 阅读 API 安装说明。
```

Comment:
During translation of Hugging Face Blog, when you find a link referring another HF blog, please make sure the URL get updated, e.g.:

```plain
https://huggingface.co/blog/unity-api
Should be changed to:
https://huggingface.co/blog/zh/unity-api
```

Reply:
zh page 404, XD

Reply:
It's fine. This makes sure we don't need to revise every linkage of a later translated blog. @chenglu What do you think?